axis_speed: 4
axis_steering: 0
stop_button: 4
align_button: 6